src/Http/BunqResponse.php

```diff
@@ -55,4 +55,14 @@ public function getPagination()
     {
         return $this->pagination;
     }
+
+    /**
+     * @param BunqResponse $bunqResponse
+     *
+     * @return static
+     */
+    public static function castFromBunqResponse(BunqResponse $bunqResponse): BunqResponse
+    {
+        return new static($bunqResponse->getValue(), $bunqResponse->getHeaders(), $bunqResponse->getPagination());
```

Comment:
According to PSR-2 standard you should make this multiline.
Also makes it a bit more readable in my opinion

```php
 return new static(
            $bunqResponse->getValue(),
            $bunqResponse->getHeaders(),
            $bunqResponse->getPagination()
        );
```

Reply:
@DennisSnijder Thanks, fixed!

Reply:
@dnl-blkv Awesome! 😄